modules/priceFloors.js

```diff
@@ -568,6 +572,7 @@ function addFieldOverrides(overrides) {
  */
 export function handleSetFloorsConfig(config) {
   _floorsConfig = utils.pick(config, [
+    'floorMin', floorMin => floorMin,
```

Comment:
With `utils.pick` you can just leave it directly as `floorMin` instead of with the function `floorMin => floorMin`

It will pick the floorMin on the config level, and since we do not need to default it, the extra use of function call is unneeded.